enum.rb

```diff
@@ -0,0 +1,53 @@
+#:nodoc:
+module Enumerable
+  def my_each
+    length.times { |index| yield(self[index]) }
+    self
+  end
+
+  def my_each_with_index
+    length.times { |index| yield(self[index], index) }
+    self
+  end
+
+  def my_select
+    array = []
+    my_each { |index| array << index if yield(index) }
+    array
+  end
+
+  def my_all?
```

Comment:
### Should have working `my_all?`  identical and returns the same thing as ruby's [`all?`](https://ruby-doc.org/core-2.6.4/Enumerable.html#method-i-all-3F). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_all.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))
``` p %w[ant bear cat].my_all?(/t/) ``` should return same results as ``` p %w[ant bear cat].my_all?(/t/) ```
``` p [1, 2i, 3.14].my_all?(Numeric) ```  should return same results as ``` p [1, 2i, 3.14].all?(Numeric) ```
``` p [nil, true, 99].my_all?```  should return same results as ```p[nil, true, 99].all? ```